### PR TITLE
update to 2.18.0

### PR DESCRIPTION
**Pull request by cbouss**
google-cloud-storage 2.18.0

**Destination channel:** main

### Links

- PKG-2710
- https://github.com/googleapis/python-storage/tree/v2.18.0
- Relevant dependency PRs:
  - https://github.com/AnacondaRecipes/google-api-core-feedstock/pull/18
  - https://github.com/AnacondaRecipes/google-resumable-media-feedstock/pull/11

### CODE REVIEW

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,18 +1,18 @@
 {% set name = "google-cloud-storage" %}
-{% set version = "2.6.0" %}
+{% set version = "2.17.0" %}
```
Comment (Jrice1317) [inline recipe/meta.yaml]:
```suggestion
{% set version = "2.18.0" %}
```
Version `2.18.0` seems to have minor changes if you want to update to the latest

Comment (Jrice1317) [reply recipe/meta.yaml]: https://github.com/googleapis/python-storage/blob/v2.18.0/setup.py

File: recipe/meta.yaml [hunk 2]
```diff
@@ -22,24 +22,24 @@ requirements:
...
-    - google-api-core >=1.31.5,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0
+    - google-auth >=2.26.1,<3.0dev
+    - google-api-core >=2.15.0,<3.0.0dev
     - google-cloud-core >=2.3.0,<3.0dev
-    - google-resumable-media >=2.3.2
+    - google-resumable-media >=2.6.0
     - requests >=2.18.0,<3.0.0dev
+    - google-crc32c >=1.0,<2.0dev
     # protobuf extra
     - protobuf <5.0.0dev
```
Comment (Jrice1317) [inline recipe/meta.yaml]:
```suggestion
    - protobuf <6.0.0dev
    run_constrained:
    - opentelemetry-api >= 1.1.0
```
If version updated to 2.18.0

**Review by Jrice1317 — APPROVED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (cbouss) [reply recipe/meta.yaml]: Updated, thanks

**Pull request merged**